.github/workflows/auto-updates.yaml

```diff
@@ -79,92 +84,47 @@ jobs:
           VENDORED_SOURCES=$(/usr/share/cargo/bin/dh-cargo-vendored-sources 2>&1 || true)
           OUTPUT=$(echo "$VENDORED_SOURCES" | grep ^XS-Vendored-Sources-Rust: || true)
           if [ -z "$OUTPUT" ]; then
+            echo "XS-Vendored-Sources-Rust is up to date. No change is needed.";
             exit 0
           fi
           sed -i "s/^XS-Vendored-Sources-Rust:.*/$OUTPUT/" debian/control
 
           echo "modified=true" >> $GITHUB_ENV
         shell: bash
+      - name: Update the fake checksum file
+        run: |
+          set -eu
+
+          # Reads the checksum of both files
+          FAKE_CHECKSUM=$(cat debian/cargo-checksum.json | jq .package)
+          CRATE_CHECKSUM=$(cat vendor_rust/vcpkg/.cargo-checksum.json | jq .package)
+          if [ $FAKE_CHECKSUM == $CRATE_CHECKSUM ]; then
+            echo "debian/cargo-checksum.json is up to date. No change is needed.";
+            exit 0
+          fi
+          # Formats in a way where we can replace the line using sed
+          CRATE_CHECKSUM="\"package\": $CRATE_CHECKSUM,"
```

Comment:
I would either use another variable, or do the string prefix directly in sed.

Reply:
Yeah, I was doing it directly in `sed`, but I thought it would get confusing. I'll rename the variable to something like `FMT_CHECKSUM`, wdyt?